packages/adapters/bitcoin/src/adapter.ts

```diff
@@ -79,6 +127,25 @@ export class BitcoinAdapter extends AdapterBlueprint<BitcoinConnector> {
       )
     )
 
+    if (accounts && accounts.length > 1) {
+      accounts = [
+        {
+          namespace: ConstantsUtil.CHAIN.BITCOIN,
+          publicKey: accounts[0]?.publicKey ?? '',
+          path: accounts[0]?.path ?? '',
+          address: accounts[0]?.address ?? '',
+          type: 'payment'
+        },
+        {
+          namespace: ConstantsUtil.CHAIN.BITCOIN,
+          publicKey: accounts[1]?.publicKey ?? '',
+          path: accounts[1]?.path ?? '',
+          address: accounts[1]?.address ?? '',
+          type: 'ordinal'
+        }
+      ]
+    }
```

Comment:
Are these accoutns guaranteed to be in this order always? 
If so, I'd add some constants for the index like `PAYMENT_ACCOUNT = 0` etc